README.md

```diff
@@ -149,6 +149,78 @@ jobs:
   how to use the Pizza command line tool and how it works with the rest of the OpenSauced
   ecosystem.
 
+# 🚀 Quickstart
+
+Get up and running with the Pizza CLI in minutes using `npx`:
+
+1. **Ensure you have Node.js installed**
+
+   Pizza CLI can be run using `npx`, which comes with Node.js. If you don't have Node.js installed, download it from [nodejs.org](https://nodejs.org/).
+
+   > NOTE
+   > For other installation methods, see the [Install](#-Install) section.
+
+2. **Generate a configuration file**
+
+   Navigate to your project directory and run:
+   ```sh
+   npx pizza@latest generate config ./ -i
+   ```
+   This will create a `.sauced.yaml` file, interactively prompting you to attribute commit emails to GitHub handles.
+
+3. **Generate CODEOWNERS file**
+
+   In your project directory, run:
+   ```sh
+   npx pizza@latest generate codeowners .
+   ```
+   This will create a `CODEOWNERS` file based on your project's git history and the `.sauced.yaml` configuration.
+
+4. **Create OpenSauced [Contributor Insight](https://opensauced.pizza/docs/features/contributor-insights/)**
+
+   After generating the CODEOWNERS file, you can create an OpenSauced Contributor Insight:
+   ```sh
+   npx pizza@latest generate insight .
+   ```
+
+5. **Explore repository insights**
+   Get metrics and insights for your repository:
+   ```sh
+   npx pizza@latest insights repo your-username/your-repo
```

Comment:
`repo` should be `repositories`; otherwise it errors that it's an invalid command
![image](https://github.com/user-attachments/assets/dc713cb4-fc2a-4e9f-b9b2-652f435a9f5a)

Reply:
Nice catch! Thanks